addons/csw/stringtable.xml

```diff
@@ -658,6 +688,18 @@
             <Russian>[CSW] Лента 20-мм гранат для ст. гранатомёта</Russian>
             <Korean>[CSW] 20mm 고속유탄발사기 탄띠</Korean>
         </Key>
+        <Key ID="STR_ACE_CSW_GMGBelt96_displayName">
+            <English>[CSW] 40mm Grenade GMG Belt 96 Count</English>
```

Comment:
Do we include ammo count like this in any other belts' displaynames? Seems weird to me.

Reply:
Ah that was done, because there a 48 count GMG belt.